Sources/MarathonCore/PackageManager.swift

```diff
@@ -176,6 +176,12 @@ internal final class PackageManager {
         return package
     }
 
+    func removeAllPackages() throws {
+        for package in addedPackages {
+            _ = try removePackage(named: package.name)
```

Comment:
Can you annotate `removePackage(named:)` with `@discardableResult` instead, so we don't have to do the `_`?